recipe/patches/CVE-2021-3520.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/lib/lz4.c b/lib/lz4.c

```

Comment:
At the top of this patch file, I would add comment and a link to the upstream issue/PR (e.g., `See: https://github.com/lz4/lz4/pull/972`) so we know where it came from and can figure out when we should drop it. 

Other than that, I think it's okay to apply this patch and rebuild the `lz4` package.